lib/matplotlib/tri/triangulation.py

```diff
@@ -0,0 +1,8 @@
+from ._triangulation import *  # noqa: F401, F403
+from matplotlib import _api
+
+
+_api.warn_deprecated(
+    "3.7",
+    message=f"importing {__name__} is deprecated. All functionality is "
+            f"available via the top-level module matplotlib.tri")
```

Comment:
This removes all the standard information in the message; you should just set `name`, `obj_type` and `alternative`.